Description: Fix build failure with glib 2.32
 where including individual glib headers is no longer allowed.
Author: Michael Biebl <biebl@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=665576
Index: loudmouth-1.4.3/loudmouth/lm-error.c
===================================================================
--- loudmouth-1.4.3.orig/loudmouth/lm-error.c	2008-10-29 21:48:15.000000000 +0100
+++ loudmouth-1.4.3/loudmouth/lm-error.c	2012-03-26 00:24:19.518939657 +0200
@@ -19,7 +19,7 @@
  */
 
 #include <config.h>
-#include <glib/gerror.h>
+#include <glib.h>
 #include "lm-error.h"
 
 /**
